src/common/components/bodies.cpp

```diff
@@ -0,0 +1,7 @@
+#include "bodies.h"
```

Comment:
Add copyright, add change this to 
`#include "common/components/bodies.h"` so that we can reduce ambiguity if we ever have another file called bodies.h

Reply:
deleted it